src/common/servlist.c

```diff
@@ -352,6 +371,14 @@ static const struct defaultserver def[] =
 	{0,			"irc.lirex.com"},
 	{0,			"irc.naturella.com"},
 	{0,			"irc.techno-link.com"},
+	 
+#ifdef USE_OPENSSL
+    {"WirelessPT", 0, 0, 0, LOGIN_SASL, 0, TRUE},
```

Comment:
This connected to the same network as the `Nixbits` ones.. and it was called `FreeThinker`